fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBDatabaseFactory.java

```diff
@@ -463,14 +464,15 @@ public long getReverseDirectoryMaxMillisPerTransaction() {
         return reverseDirectoryMaxMillisPerTransaction;
     }
 
+    // TODO: Demote these to UNSTABLE and deprecate at some point.
```

Comment:
I guess I'm not 100% clear on this, but can we deprecate them now (according to our rules)? If this is stable, I suppose we can't remove them until version 3, but I thought deprecating "should" be okay at any time.